### ISSUE TITLE
Further support for non-Japanese movies?

### ISSUE THREAD

**Issue by Still34** [DoctorD1501/JAVMovieScraper#145]
Right now it seems very limited? Even if the film is in the existing scrape database, it doesn't seem to work half of the time. For example, I know JavBus has SOME foreign films available in their database, but it never works if I just do a search there. Even if the folder/file name is exactly the codename on the site. ![(in this case, DDFBusty-16-04-30)](http://i.imgur.com/ZkPWqPl.png)

The only way to get it to scrape is to enable the option 'Provide URL Manually,' which still doesn't work properly. It downloads the wrong poster most of the time, in this case, a cropped version of the movie screenshot, as opposed to the poster shown in the search result.

**Comment by DoctorD1501** [comment]
What are the advantages of getting western movies using the javbus scraper instead of the data18 web content scraper?

**Comment by Still34** [comment]
Actually, I discovered Data 18 has much more data after posting this. But it's still having a hard time fetching the data. On JavBus it can pretty much find the film using the simple <studio>-<date> search easily while on Data18 Web Content I have to search for the whole name.

**Comment by DoctorD1501** [comment]
Yes, it's true. Data18's search is terrible. If you use the filename cleanup on the folder first, that can sometimes help as it will expand site abbreviations out (sometimes, if they are known. The list was compiled manually and is missing entries, I'm sure).

Maybe there are some improvement to the search process I can make - I'll have to think about what I can do.
